Assets/MixedRealityToolkit/_Core/BuildAndDeploy/BuildDeployPreferences.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Internal.Utilities.Editor.Setup;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities.Editor;
```

Comment:
Why remove "Setup" as it was a good clear distinction from the Editor setup code and the rest of the project?

Reply:
It wasn't removed.

Reply:
The namespace was refactored and these classes are not part of setup.